package.json

```diff
@@ -25,24 +24,29 @@
     "dist/*.js"
   ],
   "devDependencies": {
+    "@babel/core": "^7.14.5",
+    "@babel/preset-env": "^7.14.7",
     "@vue/cli-plugin-babel": "^4.5.15",
     "@vue/cli-plugin-eslint": "^4.5.15",
-    "@vue/cli-plugin-unit-jest": "^4.5.15",
     "@vue/cli-service": "^4.5.15",
+    "@vue/compiler-sfc": "^3.2.40",
     "@vue/eslint-config-prettier": "^5.0.0",
-    "@vue/test-utils": "^1.3.0",
+    "@vue/test-utils": "^2.1.0",
+    "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.1.0",
+    "babel-jest": "^27.0.0",
     "core-js": "^3.19.3",
     "cz-conventional-changelog": "^3.3.0",
     "eslint": "^5.16.0",
     "eslint-plugin-prettier": "^3.4.1",
     "eslint-plugin-vue": "^5.0.0",
     "husky": "^4.3.8",
+    "jest": "^27.0.0",
     "prettier": "^1.18.2",
     "semantic-release": "^17.4.7",
-    "vue": "^2.6.14",
-    "vue-cli-plugin-styleguidist": "^4.44.15",
-    "vue-template-compiler": "^2.6.14"
+    "vue": "^3.2.40",
+    "vue-docgen-cli": "^4.51.0",
+    "vue3-jest": "^27.0.0-alpha.1"
```

Comment:
Can we use `@vue/vue3-jest`? The namespaced ones are the newer ones.

- [ ] Investigate